cmd/podman/play_kube.go

```diff
@@ -169,7 +171,19 @@ func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
 					return errors.Wrapf(err, "Error giving %s a label", hostPath.Path)
 				}
 				break
+			case v1.HostPathFileOrCreate:
+				if _, err := os.Stat(hostPath.Path); os.IsNotExist(err) {
+					if _, err := os.OpenFile(hostPath.Path, os.O_RDONLY|os.O_CREATE, createFilePermission); err != nil {
```

Comment:
I think we need to close the file to avoid leaking the fd.

Reply:
fixed!